squidb-addons/squidb-json/squidb-json-compiler/src/com/yahoo/squidb/json/JSONPlugin.java

```diff
@@ -16,10 +15,10 @@
  */
 public class JSONPlugin extends PluginBundle {
 
-    public JSONPlugin(ModelSpec<?, ?> modelSpec, PluginEnvironment pluginEnv) {
-        super(modelSpec, pluginEnv, Arrays.asList(
-                new JSONFieldPlugin(modelSpec, pluginEnv),
-                new JSONViewPropertyPlugin(modelSpec, pluginEnv),
-                new JSONInheritedPropertyPlugin(modelSpec, pluginEnv)));
+    public JSONPlugin() {
+        super(Arrays.asList(
+                (Class<? extends Plugin>) JSONFieldPlugin.class,
```

Comment:
This cast looks a little funky here. The alternative is

```java
Arrays.<Class<? extends Plugin>>asList(
    JSONFieldPlugin.class,
    JSONViewPropertyPlugin.class,
    JSONInheritedPropertyPlugin.class
);
```

which also looks a little funky, but is probably more clear.